src/instructlab/model/train.py

```diff
@@ -317,7 +374,11 @@ def train(
     ckpt_output_dir = Path(kwargs["ckpt_output_dir"])
 
     # NOTE: If given a data_dir, input-dir is ignored in favor of existing!
-    if not data_path or data_path.strip() == DEFAULTS.DATASETS_DIR:
+    if (
+        not data_path
+        or data_path.strip() == DEFAULTS.DATASETS_DIR
+        and not kwargs.get("phased_training")
```

Comment:
`phased_training` should not be a `**kwargs` parameter.

Reply:
fixed, running to make sure this works correctly

Reply:
yes works as expected